### PR TITLE
Add charging information to docs

### PR DESCRIPTION
**Pull request by Diggsey**
[ch19592]

### CODE REVIEW

File: source/includes/_02-passfort-data-structure.html.md [hunk 1]
```diff
@@ -1,4 +1,3 @@
-
 # Data Structure
 
 PassFort structures data on individuals and company entities in a standard
```
Comment (AinsleyWhalen) [inline source/includes/_02-passfort-data-structure.html.md:3]: "individual and company entities" (no "s")

Comment (AinsleyWhalen) [reply source/includes/_02-passfort-data-structure.html.md:3]: "Additionally, many data structure types are common across different checks." 

(reword + full stop)

File: source/includes/_02-passfort-data-structure.html.md [hunk 2]
```diff
@@ -117,8 +117,53 @@ The data provider returned an error message when processing this check.
...
+        The amount to charge (in pennies). In the future, it may be possible
+        to specify a different currency unit.
+      </td>
+    </tr>
+    <tr>
+      <td><code>sku</code></td>
+      <td>string</td>
+      <td>No</td>
+      <td>
+        Stock Keeping Unit. All charges with the same SKU should cost the
```
Comment (AinsleyWhalen) [inline source/includes/_02-passfort-data-structure.html.md:152]: "Stock keeping until (SKU)."

File: source/includes/_02-passfort-data-structure.html.md [hunk 3]
```diff
@@ -117,8 +117,53 @@ The data provider returned an error message when processing this check.
...
+        same amount. For providers with a fixed cost in each country, the
+        SKU can simply be the 3-letter country code in which the check was
```
Comment (AinsleyWhalen) [inline source/includes/_02-passfort-data-structure.html.md:154]: "...the SKU can simply be the 3-letter code for the country in which the check was run"

File: source/includes/_02-passfort-data-structure.html.md [hunk 4]
```diff
@@ -117,8 +117,53 @@ The data provider returned an error message when processing this check.
...
+        run. This field can be omitted if there is no sensible value to use.
+      </td>
+    </tr>
+    <tr>
+      <td><code>reference</code></td>
+      <td>string</td>
+      <td>No</td>
+      <td>
+        The reference should be unique for each check. It should be possible
+        to use the reference to correlate the charges recorded by the provider,
```
Comment (AinsleyWhalen) [inline source/includes/_02-passfort-data-structure.html.md:164]: remove comma

File: source/includes/_03-demo-checks.html.md [hunk 5]
```diff
@@ -69,56 +81,56 @@ provider but the connection failed (due to timeouts or outages).
...
+#### `NO_MATCHES` <span style="float:right">Required</span>
 
 Your integration must return a result where the data provider could not find
 any matching records for the submitted profile.
 
-#### `ONE_NAME_ADDRESS_MATCH`
+#### `ONE_NAME_ADDRESS_MATCH` <span style="float:right">Required</span>
 
 Your integration must return a result where the data provider could find a
 match on the full name and date of birth for the submitted profile in a single
```
Comment (AinsleyWhalen) [inline source/includes/_03-demo-checks.html.md:92]: Is this meant to be "full name and address"?

File: source/includes/_05-one-time-sync-ekyc.html.md.erb [hunk 6]
```diff
@@ -385,6 +385,32 @@ app.post("/checks", (req, res) => {
...
     </tr>
+    <tr>
+      <td><code>charges</code></td>
+      <td>
+        An array containing one or more
+        <a href="#check-charges">Charges</a>
+      </td>
+      <td>No</td>
+      <td>
+        A list of charges incurred whilst running this check. Charges may only
```
Comment (AinsleyWhalen) [inline source/includes/_05-one-time-sync-ekyc.html.md.erb:396]: North Americans get super confused about the word "whilst". We should use "while" wherever possible

File: source/includes/_05-one-time-sync-ekyc.html.md.erb [hunk 7]
```diff
@@ -385,6 +385,32 @@ app.post("/checks", (req, res) => {
...
+        be returned if both of these conditions are met:
+        <ul>
+          <li>No errors occurred (ie. a result was generated).</li>
```
Comment (AinsleyWhalen) [inline source/includes/_05-one-time-sync-ekyc.html.md.erb:399]: "i.e."

File: source/includes/_05-one-time-sync-ekyc.html.md.erb [hunk 8]
```diff
@@ -385,6 +385,32 @@ app.post("/checks", (req, res) => {
...
+          <li>
+            The check is being re-sold (the commercial relationship is
```
Comment (AinsleyWhalen) [inline source/includes/_05-one-time-sync-ekyc.html.md.erb:401]: "resold"

File: source/includes/_06-one-time-callback-docver.html.md.erb [hunk 9]
```diff
@@ -568,9 +568,41 @@ def complete_check(id: UUID):
...
     </tr>
+    <tr>
+      <td><code>charges</code></td>
+      <td>
+        An array containing one or more
+        <a href="#check-charges">Charges</a>
+      </td>
+      <td>No</td>
+      <td>
+        A list of charges incurred whilst running this check. Charges may only
```
Comment (AinsleyWhalen) [inline source/includes/_06-one-time-callback-docver.html.md.erb:579]: Swap "whilst" for "while"

File: source/includes/_06-one-time-callback-docver.html.md.erb [hunk 10]
```diff
@@ -568,9 +568,41 @@ def complete_check(id: UUID):
...
+        be returned if both of these conditions are met:
+        <ul>
+          <li>No errors occurred (ie. a result was generated).</li>
+          <li>
+            The check is being re-sold (the commercial relationship is
```
Comment (AinsleyWhalen) [inline source/includes/_06-one-time-callback-docver.html.md.erb:584]: "resold"

File: source/includes/_06a-one-time-callback-docfetch.html.md.erb [hunk 11]
```diff
@@ -565,9 +565,41 @@ def complete_check(id: UUID):
...
     </tr>
+    <tr>
+      <td><code>charges</code></td>
+      <td>
+        An array containing one or more
+        <a href="#check-charges">Charges</a>
+      </td>
+      <td>No</td>
+      <td>
+        A list of charges incurred whilst running this check. Charges may only
```
Comment (AinsleyWhalen) [inline source/includes/_06a-one-time-callback-docfetch.html.md.erb:576]: "while"

File: source/includes/_06a-one-time-callback-docfetch.html.md.erb [hunk 12]
```diff
@@ -565,9 +565,41 @@ def complete_check(id: UUID):
...
+        be returned if both of these conditions are met:
+        <ul>
+          <li>No errors occurred (ie. a result was generated).</li>
+          <li>
+            The check is being re-sold (the commercial relationship is
```
Comment (AinsleyWhalen) [inline source/includes/_06a-one-time-callback-docfetch.html.md.erb:581]: "resold"

File: source/includes/_05-one-time-sync-ekyc.html.md.erb (hunk 6 above)
Comment (Diggsey) [reply source/includes/_05-one-time-sync-ekyc.html.md.erb:396]: 🤨

Comment (AinsleyWhalen) [reply source/includes/_05-one-time-sync-ekyc.html.md.erb:396]: I know, I know. But they really confused. Many of them don't know it's a word

**Pull request merged**